mmv1/products/alloydb/terraform.yaml

```diff
@@ -0,0 +1,50 @@
+# Copyright 2021 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+!ruby/object:Provider::Terraform::Config
+overrides: !ruby/object:Overrides::ResourceOverrides
+  Cluster: !ruby/object:Overrides::Terraform::ResourceOverride
+    import_format: ["projects/{{project}}/locations/{{location}}/clusters/{{cluster_id}}", "{{cluster_id}}"]
+    properties:
+      initialUser: !ruby/object:Overrides::Terraform::PropertyOverride
+        custom_flatten: "templates/terraform/custom_flatten/alloydb_cluster_input_user_flatten.go.erb"
```

Comment:
If this field is being ignored during read, do we need this custom flatten? What do we lose by removing it?

Reply:
from my tests it showed a diff otherwise that causes a recreate. it there is some setting or flag i can add to bypass this lmk